config/jobs/kubernetes/sig-cluster-lifecycle/k8s-upgrade-gce.yaml

```diff
@@ -20,7 +20,31 @@ periodics:
       - --gcp-node-image=gci
       - --gcp-zone=us-central1-f
       - --provider=gce
-      - --test_args=--kubectl-path=../../../../kubernetes_skew/cluster/kubectl.sh --minStartupPods=8
+      - --test_args=--ginkgo.focus=\[Serial\]|\[Disruptive\] --ginkgo.skip=\[Flaky\]|\[Feature:.+\] --kubectl-path=../../../../kubernetes_skew/cluster/kubectl.sh --minStartupPods=8
+      - --timeout=900m
+      - --upgrade_args=--ginkgo.focus=\[Feature:ClusterUpgrade\] --upgrade-target=ci/k8s-beta --upgrade-image=gci
+      image: gcr.io/k8s-testimages/kubekins-e2e:v20181105-66d3bcb98-master
+- interval: 2h
+  name: ci-kubernetes-e2e-gce-stable1-beta-upgrade-cluster-parallel
+  labels:
+    preset-service-account: "true"
+    preset-k8s-ssh: "true"
+  spec:
+    containers:
+    - args:
+      - --timeout=920
+      - --bare
+      - --scenario=kubernetes_e2e
+      - --
+      - --check-leaked-resources
+      - --check-version-skew=false
+      - --env=STORAGE_MEDIA_TYPE=application/vnd.kubernetes.protobuf
+      - --extract=ci/k8s-beta
+      - --extract=ci/k8s-stable1
+      - --gcp-node-image=gci
+      - --gcp-zone=us-central1-f
+      - --provider=gce
```

Comment:
We need `- --ginkgo-parallel` here I think

Reply:
That might help :man_facepalming: